tests/slack-hook-handler.spec.js

```diff
@@ -55,6 +55,31 @@ describe("SlackHookHandler.handle", function() {
             expect(intent.sendText).toHaveBeenCalledWith(
                 "!room:host", "<special & characters>");
         });
+        it("unicode-ifies one emojum", function() {
+            intent = createSpyObj("intent", ["sendText"]);
+            handler.handle(makeRequest("Lucille", ":wink:"));
+            expect(intent.sendText).toHaveBeenCalledWith("!room:host", "😉");
+        });
+        it("unicode-ifies several emoji", function() {
+            intent = createSpyObj("intent", ["sendText"]);
+            handler.handle(makeRequest("Lucille", ":wink::wink: :rugby_football:"));
+            expect(intent.sendText).toHaveBeenCalledWith("!room:host", "😉😉 🏉");
+        });
+        it("doesn't unicode-ifies emoji with whitespace", function() {
+            intent = createSpyObj("intent", ["sendText"]);
+            handler.handle(makeRequest("Lucille", ":win k:"));
+            expect(intent.sendText).toHaveBeenCalledWith("!room:host", ":win k:");
+        });
+        it("doesn't unicode-ifies improperly formed emoji", function() {
+            intent = createSpyObj("intent", ["sendText"]);
+            handler.handle(makeRequest("Lucille", ":wink:k:"));
```

Comment:
This gives the parser an easy ride since it can match in one sweep `:wink:`, we should also do `:k:wink:` to make sure that it discards that first `:` as a potential match.